vo_models/uws/models.py

```diff
@@ -65,15 +66,18 @@ def __init__(__pydantic_self__, **data) -> None:  # pylint: disable=no-self-argu
 
 
 class ErrorSummary(BaseXmlModel, tag="errorSummary", ns="uws", nsmap=NSMAP):
-    """A short summary of an error - a fuller representation of the
-        error may be retrieved from /{jobs}/{job-id}/error
+    """A short summary of an error
 
-    Elements:
-    message (str):  a short description of the error.
+    A fuller representation of the error may be retrieved from /{jobs}/{job-id}/error
 
-    Attributes:
-    type (ErrorType):   Characterization of the type of the error
-    has_detail (bool):  If true then there is a more detailed error message available at /{jobs}/{job-id}/error
+    Parameters:
+        message (str):
+            (element) - A short description of the error.
+
+        type (ErrorType):
+            (attr) - Characterization of the type of the error
+        has_detail (bool):
+            (attr) - If true then there is a more detailed error message available at /{jobs}/{job-id}/error
```

Comment:
if you haven't seen this already, you might take a look at [PEP 727](https://peps.python.org/pep-0727/), which adds documentation support to type annotations. This allows you to not have to define the type annotations twice, once as the type annotation and then again in your docstrings. It's not been accepted yet, and as far as I know, there's not a current way to get them to show up in Sphinx, but there is support for it in the [typing_extensions](https://github.com/python/typing_extensions) package and [mkdocstrings](https://github.com/mkdocstrings/mkdocstrings) for the MkDocs project. Perhaps worth an issue in the project to revisit if you don't want to deal with it now.

My understanding is that you could put the docstring helper text in as type annotations directly.

So below, for `message`, you could take out the docstring, and add to the type annotation this (maybe?):

```py
from typing import Annotated
from typing_extensions import Doc

message: Annotated[str, element, "", Doc("A short description of the error.")] = ""
```

I'm not sure I actually got the `Annotated` correct there, but there's an example in `pydantic-xml` [here](https://pydantic-xml.readthedocs.io/en/latest/pages/misc.html#optional-type-encoding)